sdk/scheduler/src/main/java/com/mesosphere/sdk/http/EndpointUtils.java

```diff
@@ -55,20 +60,32 @@ public static String toAutoIpDomain(String serviceName, SchedulerConfig schedule
      */
     public static String toAutoIpHostname(String serviceName, String taskName, SchedulerConfig schedulerConfig) {
         // Unlike with VIPs and mesos-dns hostnames, dots are converted to dashes with autoip hostnames. See DCOS-16086.
```

Comment:
It would be nice for the comment to have an example of the translation

Reply:
Also, I thought it was just /test/integration/serivce => testintegrationservice?

Reply:
I am also confused by the reference to dots

Reply:
Oh, this is a task name thing?

Reply:
Turns out autoip has different string mangling depending on whether it's the task name or the framework name